platform/metrics/platform-metrics-collector/src/test/java/ddf/metrics/collector/rrd4j/RrdJmxCollectorTest.java

```diff
@@ -142,6 +143,7 @@ public void testRrdFileCreationForDeriveDataSource() throws Exception {
     }
 
     @Test
+    @Ignore
```

Comment:
Please add a comment in the `Ignore` annotation about the problem this test is causing.

http://junit.sourceforge.net/javadoc/org/junit/Ignore.html